src/home/CallList.tsx

```diff
@@ -74,7 +74,10 @@ function CallTile({
 }: CallTileProps) {
   return (
     <div className={styles.callTile}>
-      <Link to={`/room/${roomId}`} className={styles.callTileLink}>
+      <Link
+        to={`/${roomId.substring(1).split(":")[0]}`}
```

Comment:
The fact that the variable is called `roomId` is worrying, because this approach only works for room aliases. Would you mind also renaming it to something more appropriate?